tests/test_desktop_functions.py

```diff
@@ -95,6 +95,7 @@ def test_create_and_remove_desktop():
 
     time.sleep(1) # Got to wait for the animation before we can return
```

Comment:
Do you think that could be replaced with

https://github.com/mrob95/pyvda/blob/11762ff22a2799833dbf9116edc00501d39d4cc6/pyvda/com_defns.py#L207

?

And/or overload the "future method" with `time.sleep(1)` on unsupporting platforms?

(However, I wouldn't know how to code that 😅)